docs/tutorials/query-language.md

```diff
@@ -0,0 +1,58 @@
+The Realm JavaScript SDK supports querying based on a language inspired by [NSPredicate](https://realm.io/news/nspredicate-cheatsheet/). 
+
+You query your Realm using the {@link Realm.Collection#filtered Collection.filtered()} method:
```

Comment:
All the docs might sound better if we didn't use the word `You` and were instead passive:
`The {@link Realm.Collection#filtered Collection.filtered()} method is used to query a Realm`